misc/sample_application/README.rst

```diff
@@ -0,0 +1,12 @@
+ACRN Sample Application
+#######################
+
+This directory contains the software that is necessary to run a real-time application between a real-time VM and a standard VM using the acrn-hypervisor.

```

Comment:
This description is confusing. Its not an application that is required to run apps in a RT VM and a standard VM in general. This just works on its own.
This app is rather something that can be used to forward cycletest result data from a headless RT VM to a standard VM to get them visualized.